app/formSchema/uiSchema.ts

```diff
@@ -523,8 +528,10 @@ const uiSchema = {
       'Copies of registration and other relevant documents related to incorporation, limited partnership, joint venture, not-for-profit status, etc.',
     'ui:options': {
       maxLength: MAX_LONG_INPUT_LENGTH,
-      label: false,
       allowMultipleFiles: true,
+      // Placing this custom title here as it is the top level field in
+      // supporting documents page so it will sit on top.
+      customTitle: SupportingDocuments,
```

Comment:
Not sure why these top level objects such as `supportingDocument`don't seem to pass `ui:options` but it might be worth looking into in the future. I think it's fine as is for now though.